### PR TITLE
Added source gen tests to composer

### PR DESCRIPTION
**Pull request by NipunaMarcus**

### CODE REVIEW

File: integration-tests/ui-integration-tests/src/test-setup-scripts/utils/child-process-manager.js [hunk 1]
```diff
@@ -53,5 +53,10 @@ module.exports = {
             stdio: 'ignore',
             detached: true
         }).unref();
+        
+        // spawn('node', ['src/test-setup-scripts/stop-selenium.js'], {
```
Comment (sumuditha-viraj) [inline integration-tests/ui-integration-tests/src/test-setup-scripts/utils/child-process-manager.js]: Can we remove commented codes?

File: integration-tests/ui-integration-tests/pom.xml [hunk 2]
```diff
@@ -366,39 +383,54 @@
...
-                            <executable>npm</executable>
-                            <arguments>
-                                <argument>run</argument>
-                                <argument>ui-test</argument>
-                                <argument>--</argument>
-                                <argument>--skipTests=${maven.test.skip}</argument>
-                            </arguments>
-                        </configuration>
-                    </execution>
+                    <!--<execution>-->
```
Comment (sumuditha-viraj) [inline integration-tests/ui-integration-tests/pom.xml]: Can we remove commented codes?

File: integration-tests/ui-integration-tests/pom.xml [hunk 3]
```diff
@@ -239,6 +239,23 @@
                             </resources>
                         </configuration>
                     </execution>
+                    <!--<execution>-->
```
Comment (sumuditha-viraj) [inline integration-tests/ui-integration-tests/pom.xml]: Can we remove commented codes?

File: integration-tests/ui-integration-tests/src/test-setup-scripts/utils/child-process-manager.js (hunk 1 above)
Comment (NipunaMarcus) [inline integration-tests/ui-integration-tests/src/test-setup-scripts/utils/child-process-manager.js]: I commented them as they will be needed when we have UI automation and they are still under development

File: integration-tests/ui-integration-tests/pom.xml (hunk 2 above)
Comment (NipunaMarcus) [inline integration-tests/ui-integration-tests/pom.xml]: These will be removed after reasearch on testing framework for UI tests are done

File: integration-tests/ui-integration-tests/pom.xml (hunk 3 above)
Comment (NipunaMarcus) [inline integration-tests/ui-integration-tests/pom.xml]: These will be removed after reasearch on testing framework for UI tests are done

**Pull request merged**